### ISSUE TITLE
Null pointer on latest release when running matrix job

### ISSUE THREAD

**Issue by dpricha89** [jenkinsci/slack-plugin#560]
```16:10:05 FATAL: null
16:10:05 java.lang.NullPointerException
16:10:05 	at jenkins.plugins.slack.SlackMatrixBridge$1.startBuild(SlackMatrixBridge.java:19)
16:10:05 	at hudson.matrix.DefaultMatrixExecutionStrategyImpl.notifyStartBuild(DefaultMatrixExecutionStrategyImpl.java:214)
16:10:05 	at hudson.matrix.DefaultMatrixExecutionStrategyImpl.run(DefaultMatrixExecutionStrategyImpl.java:129)
16:10:05 	at hudson.matrix.MatrixBuild$MatrixBuildExecution.doRun(MatrixBuild.java:375)
16:10:05 	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:504)
16:10:05 	at hudson.model.Run.execute(Run.java:1816)
16:10:05 	at hudson.matrix.MatrixBuild.run(MatrixBuild.java:323)
16:10:05 	at hudson.model.ResourceController.execute(ResourceController.java:97)
16:10:05 	at hudson.model.Executor.run(Executor.java:429)```

**Comment by casz** [comment]
Even though you say latest can you please provide a version number of the plugin?

**Comment by dpricha89** [comment]
v2.21

**Comment by casz** [comment]
Which is not latest :laughing: 
https://github.com/jenkinsci/slack-plugin/commit/a504b4a9b1b0816a3948b11682f770a2f18e05da

**Comment by timja** [comment]
Duplicate of https://github.com/jenkinsci/slack-plugin/issues/557 fixed in 2.22

**Comment by dpricha89** [comment]
Thank you @casz and @timja!! Just tested 2.22 and it's fixed.
